tests/test_acceptance.py

```diff
@@ -35,14 +36,13 @@ def test_slurm(tmp_path: Path, scenario: Dict):
     test_scenario_path = scenario["path"]
     expected_dirs_number = scenario.get("expected_dirs_number")
 
-    handle_dry_run_and_run(
-        "dry-run",
-        Path("conf/v0.6/general/system/example_slurm_cluster.toml"),
-        Path("conf/v0.6/general/test_template"),
-        Path("conf/v0.6/general/test"),
-        test_scenario_path,
-        tmp_path,
+    parser = Parser(
+        Path("conf/v0.6/general/system/example_slurm_cluster.toml"), Path("conf/v0.6/general/test_template")
     )
+    system, tests, test_scenario = parser.parse(Path("conf/v0.6/general/test"), test_scenario_path)
```

Comment:
**Explanation** - At this point, tests will have a list of tests that are used only in the test scenario. Therefore, we can safely check if the test templates were installed or not.

Reply:
Yes, I wanted to double check it with you, thank you for looking into it!